### PR TITLE
Add documentation for simple and advance search

### PR DESCRIPTION
**Pull request by GretaD**
### ☑️ Resolves

* Fix https://github.com/nextcloud/mail/pull/9599

### CODE REVIEW

File: user_manual/groupware/mail.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Switch layout
 1. Visit mail settings
 2. Choose between *List*, *Vertical split* and *Horizontal split*
 
-.. figure:: images/mail_layout.png
+.. figure:: ../images/mail_layout.png
```
Comment (kesselb) [inline user_manual/groupware/mail.rst:20]:
```suggestion
.. figure:: images/mail_layout.png
```

File: user_manual/groupware/mail.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ Add a new mail account
 2. Click the mail icon on the header
 3. Fill up the login form (auto or manual)
 
-.. figure:: images/new-mail-account.png
+.. figure:: ../images/new-mail-account.png
```
Comment (kesselb) [inline user_manual/groupware/mail.rst:29]:
```suggestion
.. figure:: images/new-mail-account.png
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Let's remove the screenshot because it will get outdated quicky and they contain sensitive information

**Comment by GretaD** [comment]
> Let's remove the screenshot because it will get outdated quicky and they contain sensitive information

the whole should know that we have team calls.

File: user_manual/groupware/mail.rst (hunk 1 above)
Comment (ChristophWurst) [inline user_manual/groupware/mail.rst:20]: The file is located at https://github.com/nextcloud/documentation/blob/e20930354174492c20c95dcb8ee621df3728e6a2/user_manual/groupware/images/mail_layout.png, not https://github.com/nextcloud/documentation/blob/e20930354174492c20c95dcb8ee621df3728e6a2/user_manual/images/mail_layout.png

File: user_manual/groupware/mail.rst (hunk 2 above)
Comment (ChristophWurst) [inline user_manual/groupware/mail.rst:29]: please provide screenshots. this does not render.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: user_manual/groupware/mail.rst (hunk 2 above)
Comment (ChristophWurst) [reply user_manual/groupware/mail.rst:29]: @GretaD ^

**Comment by ChristophWurst** [comment]
yes?

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/documentation/actions/runs/10079255937/job/27866159958?pr=11863

> Warning, treated as error:
> /home/runner/actions-runner/_work/documentation/documentation/user_manual/groupware/mail.rst:30:image file not > readable: images/new-mail-account.png
> make: *** [Makefile:65: html-lang-en] Error 2

**Comment by GretaD** [comment]
> https://github.com/nextcloud/documentation/actions/runs/10079255937/job/27866159958?pr=11863
> 
> > Warning, treated as error:
> > /home/runner/actions-runner/_work/documentation/documentation/user_manual/groupware/mail.rst:30:image file not > readable: images/new-mail-account.png
> > make: *** [Makefile:65: html-lang-en] Error 2

sorry, i changed the files folder and forgot to change the path

**Review by kesselb — APPROVED** [review]
LGTM :+1:

**Pull request merged**